src/test/java/com/jcabi/beanstalk/maven/plugin/DeployMojoTest.java

```diff
@@ -29,13 +29,14 @@
  */
 package com.jcabi.beanstalk.maven.plugin;
 
+import junit.framework.TestCase;
 import org.junit.Test;
 
 /**
  * Test case for {@link DeployMojo} (more detailed test is in maven invoker).
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
- * @checkstyle MultipleStringLiteralsCheck (10 lines)
+ * @checkstyle MultipleStringLiteralsCheck (100 lines)
```

Comment:
@tmjee let's use different strings in test methods instead using checkstyle's suppression